packages/ingest/queue.js

```diff
@@ -85,3 +89,41 @@ async function enqueueGranuleIngestMessage(
   return sendSQSMessage(queueUrl, message);
 }
 exports.enqueueGranuleIngestMessage = enqueueGranuleIngestMessage;
+
+/**
+ * Queue a workflow to be picked up by SF starter
+ *
+ * @param {Object} event - event to queue with workflow and payload info
+ * @returns {Promise} - resolves when the message has been enqueued
+ */
+async function queueWorkflowMessage(event) {
```

Comment:
It would be nice if we had a test for this function specifically, but wouldn't want to block this PR for that

Reply:
Nope you're right and it really wasn't a big deal to add it actually. Thanks!